### PR TITLE
Fix bug 9703 by commenting out the wrong command

### PR DESCRIPTION
**Pull request by drzymalanet**
This fixes issue https://github.com/helix-editor/helix/issues/9703 by removing the wrong formatting command for justfiles.

### CODE REVIEW

**Review by the-mikedavis — APPROVED** [review]
Thanks!

File: languages.toml [hunk 1]
```diff
@@ -2915,9 +2915,9 @@ scope = "source.just"
 file-types = [{ glob = "justfile" }, { glob = "Justfile" }, { glob = ".justfile" }, { glob = ".Justfile" }]
 injection-regex = "just"
 comment-token = "#"
-indent = { tab-width = 4, unit = "\t" }
-auto-format = true
-formatter = { command = "just", args = ["--dump"] }
+indent = { tab-width = 4, unit = " " }
```
Comment (the-mikedavis) [inline languages.toml:2918]: Is this change intentional? It will make the tab key indent by one space

Comment (drzymalanet) [reply languages.toml:2918]: Oops intended to be four spaces. Checking...

**Review by the-mikedavis — APPROVED** [review]
